From bf9561ff0d8d1b0a3ad764423c5f542fd7d291dd Mon Sep 17 00:00:00 2001
From: Hec <hec@heccraft.com>
Date: Sun, 26 Jul 2026 13:27:04 -0400
Subject: [PATCH] fix(integrations): Don't set runtime-gnu as depend

Most of the other integrations already have it set as a build-depend
and not a depend/runtime-depend, so these should too. I did not touch
the ones that are specified as a stack as stacks work inherently
differently. They should maybe not have runtime-gnu as a dependency in
the stack, though.

Without this change, vm/mesa-default.bst will break my system when
trying to use it with a different set of coreutils.

Signed-off-by: Hec <hec@heccraft.com>
---
 elements/integration/debug-app-link.bst | 2 +-
 elements/integration/ldconfig.bst       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/elements/integration/debug-app-link.bst b/elements/integration/debug-app-link.bst
index 58dd6285f..5b59272f7 100644
--- a/elements/integration/debug-app-link.bst
+++ b/elements/integration/debug-app-link.bst
@@ -4,7 +4,7 @@
 kind: manual
 description: Creates a debug directory, links the debug data of /app to a subdirectory of it
 
-depends:
+build-depends:
 - public-stacks/runtime-gnu.bst
 
 variables:
diff --git a/elements/integration/ldconfig.bst b/elements/integration/ldconfig.bst
index b42e4ac8c..896b4649f 100644
--- a/elements/integration/ldconfig.bst
+++ b/elements/integration/ldconfig.bst
@@ -3,7 +3,7 @@
 
 kind: manual
 
-depends:
+build-depends:
 - public-stacks/runtime-gnu.bst
 
 variables:
-- 
2.55.0

